gradle.properties

```diff
@@ -1,4 +1,4 @@
-VERSION_NAME=1.11.8-SNAPSHOT
+VERSION_NAME=1.11.8.1-KIG-SNAPSHOT
```

Comment:
Should this be renamed?

Reply:
Yea, this has been reverted back to what was there. There is no need to change since it only contains tests